### PR TITLE
Simplify the NF setup process by using smart cat instead of ugly sed

### PR DESCRIPTION
**Pull request by kniec**
Found a better way to create config file by directly evaluating the environment variables instead of changing them using `sed`.

### CODE REVIEW

File: content/nextflow-on-aws-batch/20_nextflow101/_index.md [hunk 1]
```diff
@@ -6,22 +6,20 @@ weight: 20
 
 To get started with Nextflow we are going to run a little example workflow locally.
 
-
 ## Clone nextflow-tutorial
 
 We will use the example Genomics workflow of the nextflow-tutorial. 
 
-```
+```bash
```
Comment (kniec) [inline content/nextflow-on-aws-batch/20_nextflow101/_index.md:13]: and doing some Markdown typo things

**Comment by kniec** [comment]
@alexis-turpin-aws, @plample small adjustment

File: content/nextflow-on-aws-batch/40_nextflow202/10_setup.md [hunk 2]
```diff
@@ -8,35 +8,36 @@ weight: 10
 
 Now that we have created queues and compute environments, we can wire them into Nextflow.
 
-{{% notice info %}}
-Please note that we are creating a config that does not hold a container name nor a AWS_REGION. We are going to change those values using `sed`.
-{{% /notice %}}
+Nextflow will evaluate a `nextflow.config` file next to the script we are executing (which would be the file in the current directory) and also fall back to `$HOME/.nextflow/config` for additional configuration. As we are going to use the latter one when using AWS Batch squared we are changing both.
```
Comment (alexis-turpin-aws) [inline content/nextflow-on-aws-batch/40_nextflow202/10_setup.md:11]: > when using AWS Batch **squared** we are changing both

Is squared a typo ? I don't get it.

Comment (kniec) [reply content/nextflow-on-aws-batch/40_nextflow202/10_setup.md:11]: lol, no - it is not. I admit it does not really contribute to make it less confusing. 
NF is supervising the execution of the workflow - no matter which executor (Batch, docker, local, K8s) is used. NF+batch-executor is the local run.
NF submitted as a batch-job itself, which submits more jobs to another queue is called batch-squared...

Comment (alexis-turpin-aws) [reply content/nextflow-on-aws-batch/40_nextflow202/10_setup.md:11]: > As we are going to use the latter one when using AWS Batch queue `batch-squared` we are changing both.

Does that sound better to you ?

**Pull request merged**